src/Interceptor/Interceptor/EventInterceptor.php

```diff
@@ -117,4 +142,14 @@ private function getEventName(Instance $instance, Event $annotation, string $typ
 
         return "{$prefix}.{$type}.{$name}";
     }
+
+    private function sendPostExecutionGenericEvent(Instance $instance): void
+    {
+        $event = $this->genericEventHandler?->make(
```

Comment:
here you can give the handler in argument and use the same one

Reply:
nope because the handler comes from the annotation see line 60:
```
$handler = $this->getHandler(EventHandler::class, $annotation);
```
and the event wont have the same properties